wire-ios/Wire-iOS Tests/StringCapitalizationTests.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import XCTest
+@testable import Wire
+
+final class StringCapitalizationTests: XCTestCase {
+
+    func testThatTheFirstLetterOfTheSentenceIsCapitalized() {
+        // GIVEN
+        let string = "hello world!"
+
+        // sanity check
+        XCTAssertEqual(string, "hello world!")
+
+        // WHEN
+        let capitalizedString = string.capitalizedSentence
+
+        // THEN
+        XCTAssertEqual(capitalizedString, "Hello world!")
```

Comment:
The implementation also lowercases all the other characters, so you should assert that in your test as well.